libpyvinyl/Parameters/Collections.py

```diff
@@ -155,6 +189,10 @@ def to_dict(self):
         params = {}
         for key in self.parameters:
             params[key] = self.parameters[key].__dict__
+            a = params[key]
+            if "_Parameter__value_type" in a:
```

Comment:
If this is exported, it becomes a mess and it wil be anyway set from the actual values in the from_dict when it is read back.